src/test/todolist.test.js

```diff
@@ -1,32 +1,38 @@
-/* eslint-disable quotes */
-import addData from "../module/addData.js";
-import clearAllData from "../module/clearAllData.js";
-import deletTasksFromLocalSorage from "../module/deleteData.js";
+import 'jest-localstorage-mock';
+import addData from '../module/addData.js';
+import clearAllData from '../module/clearAllData.js';
+import deletTasksFromLocalSorage from '../module/deleteData.js';
+import { testCurrentValue, testEditValue } from './testEditData.js';
 
-import getdata from "../module/getData.js";
-import testCheckBoxValue from "./testCheckData.js";
+import getdata from '../module/getData.js';
+import testCheckBoxValue from './testCheckData.js';
 
-describe("Test methods", () => {
-  test("should add data to the list", () => {
-    addData("read-book");
+describe('Test methods', () => {
+  test('should add data to the list', () => {
+    addData('test it');
     const data = getdata();
     expect(data.length).toBe(1);
   });
-
-  test("should check the checkbox of data from the list", () => {
+  test('should edit the data from the list', () => {
+    testCurrentValue('test it');
+    testEditValue('reading js');
+    const data = getdata();
+    expect(data[0].description).toBe('reading js');
+  });
+  test('should check the checkbox of data from the list', () => {
     testCheckBoxValue(true);
```

Comment:
👍 You've done a great job implementing the function to mock the change of task status. However, there are a couple of adjustments needed to ensure it behaves as per the project requirements:

- Currently, the mock function converts all todos' status to true, which is not in line with the project requirement. Instead, ensure that each todo's status is updated separately like the todo's status update on your webpage, when the corresponding checkbox is clicked.

- Additionally, in the `checkBoxValue` function, you used the description to update the status. Please make sure the mock function follows the same approach and updates the status based on the description.

By making these updates, your mock function will reflect the real functionality more accurately, providing a more reliable representation for testing purposes. Keep up the good work, and I'm confident that your project will work smoothly once you address these adjustments! 🌟📝🚀